common-types/src/main/scala/com/improving/app/common/errors/Validation.scala

```diff
@@ -80,37 +81,16 @@ object Validation {
     None // TODO
   }
 
-  val tenantIdValidator: Validator[TenantId] = tenantId => {
-    if (tenantId.id.isEmpty) {
-      Some(ValidationError("Tenant Id is empty"))
-    } else {
-      None
-    }
-  }
-
-  val memberIdValidator: Validator[MemberId] = memberId => {
-    if (memberId.id.isEmpty) {
-      Some(ValidationError("Member Id is empty"))
-    } else {
-      None
-    }
-  }
-
-  val organizationIdValidator: Validator[OrganizationId] = organizationId => {
-    if (organizationId.id.isEmpty) {
-      Some(ValidationError("Organization Id is empty"))
-    } else {
-      None
-    }
-  }
-
-  val storeIdValidator: Validator[StoreId] = storeId => {
-    if (storeId.id.isEmpty) {
-      Some(ValidationError("Store Id is empty"))
-    } else {
-      None
+  def endBeforeStartValidator(startTime: Option[Timestamp]): Validator[Timestamp] = endTime =>
+    startTime match {
+      case Some(startTime) =>
+        if (endTime.seconds <= startTime.seconds) {
```

Comment:
Are there issues with 0-length events we foresee?

Reply:
I think those should be validated by gateway